pylint/test/extensions/test_check_raise_docs.py

```diff
@@ -115,10 +115,14 @@ def my_func(self):
             """This is a docstring.
 
             :raises RuntimeError: Always
-            :raises NameError: Never
+            :except NameError: Never
```

Comment:
@AWhetter I saw you commented on the new test case suggesting that it might be a useless test, but it looks like you deleted the comment. Any concerns? Should I remove that test case?